static/js/search_suggestion.js

```diff
@@ -168,23 +176,28 @@ function get_group_suggestions(last, operators) {
 
     const prefix = Filter.operator_to_prefix("pm-with", negated);
 
-    const highlight_person = make_person_highlighter(last_part);
+    const person_highlighter = make_person_highlighter(last_part);
 
     const suggestions = persons.map((person) => {
         const term = {
             operator: "pm-with",
             operand: all_but_last_part + "," + person.email,
             negated,
         };
-        const name = highlight_person(person);
+        const user_pill_context = highlight_person(person, person_highlighter);
         const description =
-            prefix + " " + Handlebars.Utils.escapeExpression(all_but_last_part) + "," + name;
+            prefix + "&nbsp;" + Handlebars.Utils.escapeExpression(all_but_last_part) + ",";
```

Comment:
Would it be better to just put the `" "` inside the `Handlebars.Utils.escapeExpression`?  Seems cleaner to have `prefix + Handlebars.Utils.escapeExpression(" " + all_but_last_part + ",")`